README

```diff
@@ -294,6 +301,15 @@ The default behaviour can be changed with the following directives:
 
 	Default: 5000 (== 5 s).
 
+    LookupArgumentName
```

Comment:
That would be good but I can't think of anything short and explanatory enough.
What about LookupCertmapHintParam?